libraries/Microsoft.Bot.Connector/Teams/TeamsOperations.cs

```diff
@@ -563,6 +563,36 @@ public async Task<HttpOperationResponse<string>> SendMessageToAllUsersInTenantAs
             return result;            
         }
 
+        /// <summary>
+        /// Gets the state of a operation.
+        /// </summary>
+        /// <param name="operationId"> The operationId to get the state. </param>
+        /// <param name="customHeaders"> Headers that will be added to request. </param>
+        /// <param name='cancellationToken'> The cancellation token.  </param>
+        /// <exception cref="HttpOperationException">
+        /// Thrown when the operation returned an invalid status code.
+        /// </exception>
+        /// <exception cref="ValidationException">
+        /// Thrown when an input value does not match the expected data type, range or pattern.
+        /// </exception>
+        /// <returns>
+        /// A response object containing the state and responses of the operation.
+        /// </returns>
+        public async Task<HttpOperationResponse<string>> GetOperationStateAsync(string operationId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
We discussed that we should probably create a class to handle the response of this operation. It's not a string anymore.